l10n_br_sale_product/sale.py

```diff
@@ -251,6 +261,12 @@ def onchange_amount_costs(self, cr, uid, ids, amount_costs=False):
 class SaleOrderLine(orm.Model):
     _inherit = 'sale.order.line'
 
+    def _calc_line_base_price(self, cr, uid, line, context=None):
+        return line.price_unit * (1 - (line.discount or 0.0) / 100.0)
```

Comment:
I'm talking about this method, the code is equal to odoo core, so I don't think it needs to be redefined here.